### ISSUE TITLE
Merge conflict not solved in source code 

### ISSUE THREAD

**Issue by michmx** [hsf-training/hsf-training-matplotlib#14]
There are a couple of lines from a non-solved merge issue in lesson_check.py:
```
<<<<<<< HEAD
                         config, 'carpentry', ('hsf', 'swc', 'dc', 'lc', 'cp'))
=======
                         config, 'carpentry', ('swc', 'dc', 'lc', 'cp', 'incubator'))
>>>>>>> 14d73565c21f41fb51ca70cf7eefe8cfa73529d2
```
https://github.com/hsf-training/hsf-training-matplotlib/blob/59b1707490c5aa2cd8c97b869254c86d33c8e6ff/bin/lesson_check.py#L224

This is producing that the action `make lesson-check-all` fails.

**Comment by michmx** [comment]
Hi @klieret , I saw the issue here but I guess this must be propagated to other repositories.

**Comment by amangoel185** [comment]
This got fixed in #20!

**Comment by michmx** [comment]
Ok, thanks! I am closing this issue. And thanks @klieret  for the fix.
